src/main/java/org/cloudfoundry/promregator/PromregatorApplication.java

```diff
@@ -52,8 +55,13 @@
 // Warning! This implies @ComponentScan - and we really must have that in place, e.g. due to JMS :(
 
 @EnableScheduling
-@Import({ BasicAuthenticationSpringConfiguration.class, SecurityConfig.class, ErrorSpringConfiguration.class, JMSSpringConfiguration.class, AuthenticatorSpringConfiguration.class })
+@Import({ BasicAuthenticationSpringConfiguration.class,
+	SecurityConfig.class,
+	ErrorSpringConfiguration.class,
+	JMSSpringConfiguration.class,
+	AuthenticatorSpringConfiguration.class })
 @EnableAsync
+@EnableConfigurationProperties({PromregatorConfiguration.class, PromregatorConfigurationV2.class})
```

Comment:
Note the new `PromregatorConfigurationV2`, in the PR I'll be removing the old `PromregatorConfiguration` and all the `@Value` usages.